pache/lru_cache.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
+
+"""
+QNode -> holds key and value; as well as pointers to previous and next nodes.
+"""
+
+
+class DQNode(object):
+    def __init__(self, key: object, value: object):
+        """
+        Initialize the Dequeue Node.
```

Comment:
Also, let's update the comment to reflect  the rename to `PacheItem`

Reply:
Renamed the class. 0f3b0a69cd7334bdc1393dec0821b730f7889a57